Project Management Process.md

```diff
@@ -0,0 +1,293 @@
+
+#### Sprint Administrator
+
+The sprint administrator (normally [@RichardLitt](//github.com/RichardLitt)) is responsible for the sprint process every week.
+
+** GitHub tasks**
+
+- Opens a new issue for the sprint on GitHub, and posts a link to it on IRC ahead of time. To open an Etherpad, go to https://etherpad-mozilla.org.
+- Opens an etherpad for each discussion and copies in the [sprint-issue-template](sprint-issue-template.md) to each etherpad, making sure that the etherpad URL matches the discussion link in the new sprint issue.
+- Pings each of the discussion leads to remind them to prepare for their talks the next day, preferably by writing an agenda
+- Asks people to drop their updates in the old sprint issue before the sync
+- Reminds people to drop their TODOs in the new sprint issue after the discussions.
+
+** Sync tasks**
+
+- Begins each sprint sync with a roll-call by pinging active contributors (listed below) on IRC.
+- Prompts everyone who participated in the previous sprint to update on their work. The best way to choose who goes first is to go off of a first-post-first-sync method, where all participants add their updates to the old PM issue, and the first to do so generally syncs first in the IRC channel.
+- Closes up by making sure everyone who needs to has gone.
+
+** Discussion tasks**
+
+- Sets up the videos and moderates them, using the process outlined in [hangouts.md](hangouts.md).
+
+## Structure
+
+### Overview
+The project information across the organization is tracked with the structure described in this chapter. At the highest level, we have an Organization Roadmap, which lists all projects and their milestones. Each project contains a set of milestone and milestones contain one or more goals. Goals are the basic unit of work and they describe everything from bugs and new features to general tasks. Goals are collected and tracked in a Backlog.
+
+```
+Organization        |   Project 1           Project 2
+                    |  
+Roadmap             |   Roadmap             Roadmap  
+                    |  
+  Milestones        |     Milestone 1         Milestone A
+    Milestone 1     |       Goal 1              Goal A
+    Milestone 2     |       Goal 2            Milestone B
+    Milestone 3     |       Goal 3              Goal B
+    Milestone A     |     Milestone 2           Goal C
+    Milestone B     |       Goal 4              Goal D
+    Milestone C     |     Milestone 3      
+                    |       Goal 5      
+  Projects          |  
+    Project 1       |   Backlog             Backlog
+    Project 2       |     Goal 1              Goal A
+    Project 3       |     Goal 2              Goal B
+                    |     Goal 3              Goal C
+                    |     Goal 4              Goal D
+                    |     Goal 5
+```
```

Comment:
- Project 3 is implicit, it doesn't need to be show because it doesn't fit. think of it like a series `A_1, A_2, A_3, .... A_n`
- Numbers and letters are there to show that projects can identify their milestones how they wish, instead of imposing some structure from above. (this is to avoid implying numbers are a MUST and get into pedantic disagreement)
- Milestones are under the "Oranization Roadmap" to show that organizations _contain_ all projects _and their milestones_. Think of it as `ls org/projects/*/milestones/*`
- For simplicity here, i _think_ goals are always in one milestone, but i think that's a good point. goals _could_ be in more than one milestone. i think in practice mechanics may have to twist too much to support that, so i think it may be good to either _agree they NEVER belong to more than 1 milestone_ (i think overly strict) or _define how to deal with a goal needing to be in two milestones (eg list it under the one expected to complete sooner, or make a meta-issue if you have to, or something)_.